components/textbox/textbox_links.jsx

```diff
@@ -143,3 +146,9 @@ export default class TextboxLinks extends React.Component {
         );
     }
 }
+
+const mapStateToProps = (state) => ({
+       isMarkDownPreviewEnabled: Utils.isFeatureEnabled(PreReleaseFeatures.MARKDOWN_PREVIEW, state),
```

Comment:
Really minor nit, but markdown is capitalized as "Markdown", so this would be
```suggestion
       isMarkdownPreviewEnabled: Utils.isFeatureEnabled(PreReleaseFeatures.MARKDOWN_PREVIEW, state),
```